javascript/ql/src/Security/CWE-117/LogInjection.ql

```diff
@@ -4,15 +4,15 @@
  *              insertion of forged log entries by a malicious user.
  * @kind path-problem
  * @problem.severity error
- * @precision high
+ * @precision medium
```

Comment:
On line 2, please use sentence case as per [this link](https://github.com/github/codeql/blob/8d4f7e2db7ec3b7559a3e358fd1b8a7203ce7474/docs/query-metadata-style-guide.md#query-name-name).

Reply:
Oops please ignore this comment for now, I've just started a review (or at least, that's what I tried to do!).